app/src/main/res/values-de/strings.xml

```diff
@@ -7,8 +7,10 @@
     <string name="section_about">Info</string>
     <string name="section_chat">Chat</string>
     <string name="section_settings">Einstellungen</string>
+    <string name="observer_mode">Beobachtermodus</string>
     <string name="open_drawer">Menü öffnen</string>
     <string name="close_drawer">Menü schließen</string>
+    <string name="nav_observer_mode_text">Wenn du nicht selbst teilnimmst, aber sehen möchtest wo die Mass ist, aktiviere den Beobachtermodus.</string>
```

Comment:
", aktiviere den Beobachtermodus." is a little superfluous if activated. Here two new suggestions how to make it sound a little smoother:

"Kein Radfahrer, aber möchtest die Critical Mass beobachten?"
"Not riding, but still want to follow the Critical Mass?"

Reply:
You're right - in general the german texts could use some refinement. Since this is best done over at transifex to keep merging simple, I'll merge this now and we can pull updated translations before the next release.